lib/nerves_hub_link/client.ex

```diff
@@ -181,6 +181,7 @@ defmodule NervesHubLink.Client do
 
       {:ok, 0, _message} ->
         initiate_reboot()
+        NervesHubLink.send_update_status("rebooting_after_update")
```

Comment:
You may want to swap these. You're going to be racing to deliver this message before the shutdown initiates. I can't remember if this is a graceful reboot or not, but may want all the time you can't get